p4pillon/config_reader.py

```diff
@@ -116,12 +116,18 @@ def process_config(pvname: str, pvdetails: dict[str, Any]) -> BasePVRecipe:
     else:
         pvrecipe = PVScalarRecipe(PVTypes[pvdetails["type"]], pvdetails["description"], initial)
 
-    supported_configs = [("read_only", bool), ("calc", dict)]
-    for config, config_type in supported_configs:
-        # Process variables in the configuration that are attributes of the pvrecipe class
+    # Set read only
+    temp_config = pvdetails.get("read_only")
+    if temp_config is not None and isinstance(temp_config, bool):
+        pvrecipe.read_only = temp_config
+
+    # Process configuration in the yaml specific to a supported rule
+    # and add this to pvrecipe.rule_configs
+    rule_configs = [("calc", dict), ("cps_write", dict)]
```

Comment:
There shouldn't be references to rules which don't exist in p4pillon in the p4pillon code! This indicates that there needs to be an alternate way to handle this, which isn't a growing list of special cases.